### DISCUSSION TITLE
Curl Command Throws an Error

### DISCUSSION THREAD

**Discussion by rmd84** [lightningnetwork/lnd#6031] [Troubleshooting]
When running the "curl https://raw.githubusercontent.com/lightningnetwork/lnd/master/scripts/keys/guggero.asc | gpg --import" throws an error "gpg: key 8E4256593F177720: 1 signature not checked due to a missing key"

Any Help?

![image](https://user-images.githubusercontent.com/10411960/143669684-65d529b2-41b2-49cc-8361-f42739e58735.png)

**Comment by guggero** [comment]
That's not really an error, just a hint. It means, my key was signed by another key (see https://en.wikipedia.org/wiki/Web_of_trust), of which you don't have the public key to verify. You can download that other key with `curl https://jonathancross.com/C0C076132FFA7695.asc | gpg --import`. But then you get into a recursive problem since that key again is signed by other keys you don't have.

So you can safely ignore the "not checked due to a missing key" hint.
